hq/app/aws/support/TrustedAdvisor.scala

```diff
@@ -120,4 +120,10 @@ object TrustedAdvisor {
       resourcesSuppressed = result.getResult.getResourcesSummary.getResourcesSuppressed
     )
   }
+
+  def refreshTrustedAdvisorCheck(client: AWSSupportAsync, checkId: String)(implicit ec: ExecutionContext): Attempt[RefreshTrustedAdvisorCheckResult] = {
```

Comment:
There's a `getTrustedAdvisorChecks` function higher up in this Object. This function feels like the counterpart to that. As a result, I think this should be named with a plural and it should be placed alongside that function.